storage/src/journal/fixed.rs

```diff
@@ -277,6 +293,11 @@ impl<B: Blob, E: Storage<B>, A: Array> Journal<B, E, A> {
         Ok(())
     }
 
+    /// Return the position of the oldest item in the journal that remains readable.
```

Comment:
Like you noted for `forget`, I think its worth noting this could be less than `min_item_position`.

Reply:
not sure I understand, there is no min_item_position here?  You mean from a previous call to forget?

Reply:
added.